ui/panels-plugin/src/plugins/gauge-chart/GaugeChartPanel.tsx

```diff
@@ -12,27 +12,17 @@
 // limitations under the License.
 
 import { GaugeSeriesOption } from 'echarts';
-import { JsonObject } from '@perses-dev/core';
-import { GraphQueryDefinition, useGraphQuery, PanelProps } from '@perses-dev/plugin-system';
-import { GaugeChart, GaugeChartData, UnitOptions } from '@perses-dev/components';
+import { useGraphQuery, PanelProps } from '@perses-dev/plugin-system';
+import { GaugeChart, GaugeChartData } from '@perses-dev/components';
 import { Skeleton } from '@mui/material';
 import { useMemo } from 'react';
-import { convertThresholds, defaultThresholdInput, ThresholdOptions } from '../../model/thresholds';
-import { CalculationsMap, CalculationType } from '../../model/calculations';
+import { convertThresholds, defaultThresholdInput } from '../../model/thresholds';
+import { CalculationsMap } from '../../model/calculations';
 import { useSuggestedStepMs } from '../../model/time';
-
-export const GaugeChartKind = 'GaugeChart' as const;
+import { GaugeChartOptions } from './gauge-chart-model';
 
 export type GaugeChartPanelProps = PanelProps<GaugeChartOptions>;
 
-interface GaugeChartOptions extends JsonObject {
-  query: GraphQueryDefinition;
-  calculation: CalculationType;
-  unit?: UnitOptions;
-  thresholds?: ThresholdOptions;
-  max?: number;
-}
```

Comment:
Again, dope how this file got simper as all the type concerns move into the model file!